frontend/src/features/public-form/components/FormEndPage/FormEndPage.stories.tsx

```diff
@@ -29,7 +29,7 @@ Default.args = {
   },
   submissionData: {
     id: 'mockSubmissionId',
-    timeInEpochMs: 1648545566989,
+    timestamp: new Date(1648545566989),
```

Comment:
Change this back to number as well?

Reply:
oops! thanks, [dcf2dab](https://github.com/opengovsg/FormSG/pull/5672/commits/dcf2dab6e79770e81e3c54afa9194b445862d347)